crates/wasmtime/src/trampoline/func.rs

```diff
@@ -56,30 +58,37 @@ unsafe extern "C" fn stub_fn(
         Err(panic) => wasmtime_runtime::resume_panic(panic),
     }
 
-    unsafe fn call_stub(
+    unsafe fn call_stub<F>(
```

Comment:
I think it would be a bit cleaner to inline this function given how small it is.